all-spark-notebook/Dockerfile

```diff
@@ -50,3 +50,4 @@ RUN conda install --quiet --yes 'spylon-kernel=0.4*' && \
     rm -rf "/home/${NB_USER}/.local" && \
     fix-permissions "${CONDA_DIR}" && \
     fix-permissions "/home/${NB_USER}"
+    
```

Comment:
```suggestion
```

Reply:
Above was to deal with newline at end of the Dockerfile to revert it back perfectly so this file is no longer part of this PR.